README.md

```diff
@@ -58,26 +58,29 @@
-[![MochaJS Backer](https://opencollective.com/mochajs/sponsor/19/avatar)](https://opencollective.com/mochajs/sponsor/19/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/0/avatar)](https://opencollective.com/mochajs/sponsor/0/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/1/avatar)](https://opencollective.com/mochajs/sponsor/1/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/2/avatar)](https://opencollective.com/mochajs/sponsor/2/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/3/avatar)](https://opencollective.com/mochajs/sponsor/3/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/4/avatar)](https://opencollective.com/mochajs/sponsor/4/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/5/avatar)](https://opencollective.com/mochajs/sponsor/5/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/6/avatar)](https://opencollective.com/mochajs/sponsor/6/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/7/avatar)](https://opencollective.com/mochajs/sponsor/7/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/8/avatar)](https://opencollective.com/mochajs/sponsor/8/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/9/avatar)](https://opencollective.com/mochajs/sponsor/9/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/10/avatar)](https://opencollective.com/mochajs/sponsor/10/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/11/avatar)](https://opencollective.com/mochajs/sponsor/11/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/12/avatar)](https://opencollective.com/mochajs/sponsor/12/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/13/avatar)](https://opencollective.com/mochajs/sponsor/13/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/14/avatar)](https://opencollective.com/mochajs/sponsor/14/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/15/avatar)](https://opencollective.com/mochajs/sponsor/15/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/16/avatar)](https://opencollective.com/mochajs/sponsor/16/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/17/avatar)](https://opencollective.com/mochajs/sponsor/17/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/18/avatar)](https://opencollective.com/mochajs/sponsor/18/website)
+[![MochaJS Sponsors](https://opencollective.com/mochajs/sponsor/19/avatar)](https://opencollective.com/mochajs/sponsor/19/website)
+[![MochaJS Sponsors](./docs/images/jetbrains-logo.png)](https://www.jetbrains.com/)
+[![MochaJS Sponsors](./docs/images/wallabyJs-logo.png)](https://wallabyjs.com/)
+[![MochaJS Sponsors](./docs/images/saucelabs-logo.png)](https://saucelabs.com/)
```

Comment:
Separeting these logos from opencollective is better.

Reply:
Okay. I will separate it, thank you :)